packages/compiler/test/test-host.ts

```diff
@@ -37,13 +41,11 @@ class TestHostError extends Error {
   }
 }
 
-function resolveVFsPath(path: string) {
-  return resolvePath("/", path);
+export function resolveVirtualPath(path: string, ...paths: string[]) {
+  return resolvePath(process.platform === "win32" ? "Z:/test" : "/test", path, ...paths);
```

Comment:
what was the reasoning behind that change? then entire system should have already been os agnostic so shouldn't really have to do that

Reply:
The immediate reason was we're calling pathToFileURL and it throws on windows on "/something". Unfortunately, assuming "/" is an absolute path is not really "OS-agnostic". It's a current-drive relative path on Windows.

Reply:
Hmm, that's not what's throwing, that resolves to file://C:/something. I need to revert and check what the error was. On it.

Reply:
I see yeah seems like that's the only way if we have to use `pathToFileURL`. However is the one causing issue the one defined in the test host? in that case couldn't we just say instead that the url is `file://${path}`

Reply:
So, it was actually the opposite, I used file:/// as the root uri since the root of virtual file system was "/" and then URLToFilePath blew up. I think it's easier to just have the virtual root have a drive letter, but I'll add a comment to clarify why.